tests/Common/ConfigurationRowTest.php

```diff
@@ -30,6 +30,41 @@ public function setUp()
         }
     }
 
+    public function testConfigurationRowUpdateChangeDescription()
```

Comment:
Přijde mi zbytečné na to dělat novou metodu a dělat celý bootstrap jen kvuli otestování change description. Přidal bych to k existujícímu

Reply:
\Keboola\Test\Common\ComponentsTest::testComponentConfigRowUpdate